### ISSUE TITLE
Can I train modified yolo network without pretrained model?

### ISSUE THREAD

**Issue by HayeonP** [AlexeyAB/darknet#4838]
Hi, I'm trying to find the relationship between network model structure and accuracy. So I need to change the original yolo network ( layer depth, filter numbers, and so on) and train it. But I need a kind of pretrained model for training as I know. For example, I need to use pretrained model "darknet53.conv.74" when I training yolov3 network for detecting custom object.

Can I train modified yolo network without pretrained model?
If I cannot, how can I make pretrained model appropriate to modified yolo network?

**Comment by WongKinYiu** [comment]
Yes, you can.

All you need is training more epochs if your dataset is large enough (> 10k images).
![image](https://user-images.githubusercontent.com/12152972/74220458-a2773100-4cea-11ea-84cd-5e36ac75180a.png)
《Rethinking ImageNet Pre-training》

**Comment by HayeonP** [comment]
Thanks a lot! It will be very helpful to me
By the way, I have one more question. Why the graphs are divided to several line?
It means,
![image](https://user-images.githubusercontent.com/36179147/74299963-a14d0f00-4d91-11ea-8aa5-91215739827f.png)
What is the meaning of this shape?
Is it means stop and retraining the network with mid-trained weight?

**Comment by WongKinYiu** [comment]
https://arxiv.org/pdf/1811.08883.pdf
Please check  **https://arxiv.org/pdf/1811.08883.pdf** in Section 4 and **Baselines with GN and SyncBN** in Section 5.

**Comment by mahaoyanghb** [comment]
@WongKinYiu 
the original paper of YOLO make trainning into two parts: 
Part 1: choose a backbone classifer network, such as Alexnet,GoogleNet,Resnet
and make dataset for this backbone, and train it until get ideal performance.
Part 2: modify classifer backbone structure, mainly the last several layers, add regression layer for object detection, then labelling picture with bouding box , then train detection network with classifier pretrained weights(obtained from Part 1) 

you said just use the following command to train detection network from stretch.
darknet detector train data/obj.data yolov3-tiny-obj.cfg
can it work well ?
as far as I know object detection contain two part: feature extraction and location regression, 
with a feature extraction finished pretrained weight, location regression can be trained better, otherwise these two stage will do impact to each other, make trainning bad, can't get ideal result.

**Comment by WongKinYiu** [comment]
Hello, If your dataset is big enough, then you need not to use pre-trained model.
But you need train more epochs, usually 4x to 6x than using pre-trained model.
